pkg/apis/config/v1alpha1/mesh_config.go

```diff
@@ -26,8 +26,8 @@ type SidecarSpec struct {
 	EnablePrivilegedInitContainer bool   `json:"enablePrivilegedInitContainer,omitempty" yaml:"enablePrivilegedInitContainer,omitempty"`
 	LogLevel                      string `json:"logLevel,omitempty" yaml:"logLevel,omitempty"`
 	EnvoyImage                    string `json:"envoyImage,omitempty" yaml:"envoyImage,omitempty"`
-	MaxDataPlaneConnections       int    `json:"maxMaxPlaneConnections,omitempty" yaml:"max_data_plane_connections,omitempty"`
-	ConfigResyncInterval          string `json:"configResyncInterval,omitempty" yaml:"config_resync_interval,omitempty"`
+	MaxDataPlaneConnections       int    `json:"maxDataPlaneConnections,omitempty" yaml:"maxDataPlaneConnections,omitempty"`
```

Comment:
Yaml tags are not required because the k8s API operates with JSON. `kubectl` converts YAML to JSON before making a request to the API server. Kindly remove the redundant tags since they seem unnecessary.